cache/remotecache/v1/spec.go

```diff
@@ -27,16 +27,23 @@ type LayerAnnotations struct {
 }
 
 type CacheRecord struct {
-	Results []CacheResult  `json:"layers,omitempty"`
-	Digest  digest.Digest  `json:"digest,omitempty"`
-	Inputs  [][]CacheInput `json:"inputs,omitempty"`
+	Results     []CacheResult  `json:"layers,omitempty"`
+	DiffResults []DiffResult   `json:"diffs,omitempty"`
```

Comment:
One naming solution might be that if the old ones are called "layers" and only have one index then new ones are "chains" as they have a chain of indexes.

Reply:
At least in JSON I'd prefer just `json:"chains,omitempty"` . The results themselves are not chained. It's if a result points to layer index or a chain of layer indexes.

Reply:
Updated to `chains`